lib/src/openApiToZod.ts

```diff
@@ -146,6 +148,15 @@ export function getZodSchema({ schema, ctx, meta: inheritedMeta, options }: Conv
                 return code.assign("z.never()");
             }
 
+            if (schema["x-enumNames"]?.length === schema.enum?.length) {
```

Comment:
I didn't know about this `x-enumNames`, from what I've found it looks like it's coming from NSwag ? 
could you add a comment with a link to the spec reference so that we know this one is special

Reply:
Eh — both NSwag and NJsonSchema (used by NSwag) have pretty terrible documentation, included a reference to the closest thing to a specification I could [find](https://github.com/RicoSuter/NJsonSchema/wiki/Enums#enum-names-and-descriptions).